pom.xml

```diff
@@ -24,8 +24,8 @@
   <name>SFTP Actions</name>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>6.0.0-SNAPSHOT</cdap.version>
-    <hydrator.version>2.2.0-SNAPSHOT</hydrator.version>
+    <cdap.version>6.3.0-SNAPSHOT</cdap.version>
```

Comment:
actually, it might be better to use a released version like 6.2.0. Otherwise this will fail again when 6.3.0 is released. Unless we need any APIs from 6.3.0-SNAPSHOT

Reply:
changed to 6.2.9